rubicon_ml/client/dataframe.py

```diff
@@ -43,14 +43,22 @@ def get_data(self, df_type="pandas"):
         """
         project_name, experiment_id = self.parent._get_identifiers()
 
-        self._data = self.repository.get_dataframe_data(
-            project_name,
-            self.id,
-            experiment_id=experiment_id,
-            df_type=df_type,
-        )
-
-        return self._data
+        for repo in self.repositories:
+            self._data = None
+            try:
+                self._data = repo.get_dataframe_data(
+                    project_name,
+                    self.id,
+                    experiment_id=experiment_id,
+                    df_type=df_type,
+                )
+            except Exception as err:
+                return_err = err
+                pass
```

Comment:
we don't need the `pass`es in all of these - I just had it in the example in the issue cause there wasn't any other code in the `except` block